### PR TITLE
project: mozilla iot gateway and things demo

### PR DESCRIPTION
**Pull request by ric96**
Signed-off-by: Sahaj Sarup <sahaj.sarup@linaro.org>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,107 @@
...
+    $ cd ~/gateway
+    $ npm start
+    ```
+
+## 2.3) Arduino/Sensors Mezzanine Setup
+  - Setup [adafruit neopixel library](https://learn.adafruit.com/adafruit-neopixel-uberguide/arduino-library-installation#install-adafruit-neopixel-via-library-manager-13-2)
+  - In case of Arduino, simply use the IDE to flash the ```src/neo.ino``` file.
+  - For the SeeedStudio Sensors Mezzanine follow [this guide](https://www.96boards.org/blog/arduino-ide-sensor-mezzanine/) to setup and flash using the Arduino IDE.
+
+## 2.4)
```
Comment (Mani-Sadhasivam) [inline README.md]: Title?

File: README.md [hunk 2]
```diff
@@ -1 +1,107 @@
...
+## 2.1) Prerequisites
+  - Make sure the latest Debian is installed on both the 96Boards CE device. You will be able to find the instructions on the [documentation](https://www.96boards.org/documentation/consumer/) section of our website.
+  - In the case of [Dragonboard410c](https://www.96boards.org/product/dragonboard410c/) you also have the option to install [Fedora](https://nullr0ute.com/2017/11/getting-started-with-fedora-on-the-96boards-dragonboard/)
+  - Install the [Arduino IDE](https://www.arduino.cc/en/Main/Software) on your host system.
+
+## 2.2) Gateway Setup
+  - Connect your Gateway device to your home Wifi network. You can do it from the terminal using ```$ nmtui```
+  - Follow the Gateway installation Guide over at [Mozilla-IoT GitHub Repository](https://github.com/mozilla-iot/gateway).
+  - Setup thing-url-adapter addon
+    - ```
```
Comment (Mani-Sadhasivam) [inline README.md]: remove hyphen here. It produces weird rendering.

File: src/neo.ino [hunk 3]
```diff
@@ -0,0 +1,38 @@
+#include <Wire.h>
```
Comment (Mani-Sadhasivam) [inline src/neo.ino:1]: Copyright?

File: src/things.py [hunk 4]
```diff
@@ -0,0 +1,160 @@
+from webthing import (Action, Event, Property, SingleThing, Thing, Value,
```
Comment (Mani-Sadhasivam) [inline src/things.py:1]: Copyright?

File: README.md [hunk 5]
```diff
@@ -1 +1,107 @@
...
+
+- [1) Hardware](#1-hardware)
+   - [1.1) Hardware Requirements](#11-hardware-requirements)
+   - [1.2) Hardware Setup](#12-hardware-setup)
+- [2) Software](#2-software)
+   - [2.1) Build Environment Setup](#21-build-environment-setup)
+   - [2.2) API-Keys](#22-api-keys)
+   - [2.3) Running the Application](#23-running-the-application)
+- [3) Video Demonstration](#5-video-demonstration)
+- [4) Change Log](#6-change-log)
```
Comment (Mani-Sadhasivam) [inline README.md]: Are you using it?

**Review by Mani-Sadhasivam — CHANGES_REQUESTED** [review]
Minor nitpicks...

File: src/neo.ino (hunk 3 above)
Comment (ric96) [reply src/neo.ino:1]: adding LICENSE file instead

**Review by Mani-Sadhasivam — APPROVED** [review]

**Pull request merged**